app/src/main/java/org/ole/planet/myplanet/ui/teams/TeamsAdapter.kt

```
@@ -36,11 +39,38 @@ class TeamsAdapter(
+                        .circleCrop()
+                        .into(teamPhoto)
+                } else {
+                    try {
+                        val uri = Uri.parse(team.profileImage)
+                        Glide.with(root.context)
+                            .load(uri)
+                            .placeholder(R.drawable.ole_logo)
+                            .error(R.drawable.ole_logo)
+                            .circleCrop()
+                            .into(teamPhoto)
+                    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)